docs/en/sql-reference/functions/json-functions.md

```diff
@@ -552,31 +800,67 @@ text
 └────────────────────────────────────────────────────────────┘
 ```
 
-## JSONExtractRaw(json\[, indices_or_keys\]...)
+### JSONExtractRaw
 
-Returns a part of JSON as unparsed string.
+Returns part of the JSON as an unparsed string. If the part does not exist or has the wrong type, an empty string will be returned.
 
-If the part does not exist or has a wrong type, an empty string will be returned.
+**Syntax**
 
-Example:
+```sql
+JSONExtractRaw(json [, indices_or_keys]...)
+```
+
+**Parameters**
+
+- `json` — JSON string to parse. [String](../data-types/string.md).
+- `indices_or_keys` — A list of zero or more arguments each of them can be either string or integer. [String](../data-types/string.md), [Int*](../data-types/int-uint.md).
+
+`indices_or_keys` type:
+- String = access object member by key.
+- Positive integer = access the n-th member/key from the beginning.
+- Negative integer = access the n-th member/key from the end.
+
+**Returned value**
+
+- Returns part of the JSON as an unparsed string. Otherwise, an empty string is returned if the part does not exist or has the wrong type. [String](../data-types/string.md).
```

Comment:
"Otherwise" is not needed:
_If the part does not exist or has the wrong type, an empty string is returned._